src/app/pages/quickMaps/quickMaps.component.ts

```diff
@@ -32,6 +44,21 @@ export class QuickMapsComponent implements OnInit {
     });
   }
 
+  ngAfterViewInit(): void {
+    if (this.featureFlagsService.isEnabled('quick-maps-loading-tour')) {
+      setTimeout(() => {
+        const hasSeenTour = Boolean(localStorage.getItem('has-viewed-tour'));
+        console.log(hasSeenTour);
+        if (!hasSeenTour || this.featureFlagsService.isEnabled('quick-maps-tour-always')) {
```

Comment:
Once viewed it sets a localstorage flag so won't show again unless the `quick-maps-tour-always` feature flag is active (it currently is), in which case it shows every time